src/ThreadPool/ThreadPool/IMyTask.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ThreadPool;
+
+/// <summary>
+/// Interface describing task
+/// </summary>
+public interface IMyTask<TResult>
+{
+    bool IsCompleted {get; set;} 
```

Comment:
```suggestion
    bool IsCompleted { get; set; } 
```
По стайлгайду скобки отделяются пробелами тут